org.nuspell.Nuspell.json

```diff
@@ -0,0 +1,72 @@
+{
+  "app-id": "org.nuspell.Nuspell",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "nuspell",
+  "modules" : [
+    {
+      "name": "boost",
+      "buildsystem": "simple",
+      "build-commands": [
+        "./bootstrap.sh --prefix=/app --with-libraries=locale",
+        "./b2 install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.65.1/source/boost_1_65_1.tar.bz2",
+          "sha256": "9807a5d16566c57fd74fb522764e0b134a8bbe6b6e8967b83afefd30dcd3be81"
+        }
+      ]
+    },
+    {
+      "name": "launcher",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/share/applications/",
```

Comment:
Can you remove this and next line? With these gone, it's good to merge.

Reply:
Thanks, fixed with 355bfeb, please trigger build bot again.